### DISCUSSION TITLE
Multiple Classes In .hxc Files

### DISCUSSION THREAD

**Discussion by lemz1** [FunkinCrew/Funkin#3065] [Q&A] [answered]
Does anyone know if there is a way to use multiple classes in a .hxc file?

Or some way to import classes from other scripts?

**Comment by AbnormalPoof** [comment, marked as answer]
You can, here's an example:

```haxe
import funkin.play.stage.Stage;
import funkin.play.PlayState;
import funkin.audio.FunkinSound;
import funkin.graphics.FunkinSprite;
import funkin.Paths;
import flixel.tweens.FlxTween;
import flixel.tweens.FlxEase;
import flixel.FlxG;
import funkin.Highscore;
import flixel.util.FlxTimer;

class ExpoStage extends Stage {
}

class VocaStage extends Stage {
}
```

**Reply by lemz1** [reply]
ooh, i didnt know the class is forced to extend something.

thanks.

although i did notice when i made a class that extends FunkinSprite, i cant call makeSolidColor and stuff.
probably because it turned it into a ScriptedFunkinSprite, i wonder if there is a way to circumvent that.

well i guess instead of extending FunkinSprite one could do something like
```haxe
// Needs to extend something, else you won't be able to use it
class CoolSprite extends FlxBasic {
   public var spr:FunkinSprite;
   
   public function new() {
     super();
   }
}
```
